lib/Cake/Network/CakeSocket.php

```diff
@@ -116,6 +117,32 @@ class CakeSocket {
  */
 	public function __construct($config = array()) {
 		$this->config = array_merge($this->_baseConfig, $config);
+
+		// These TLS versions are not supported by older PHP versions,
```

Comment:
The whole TLS related code could be moved into a protected method in which the doc blcok could epxlain the situation.

 + smaller method
 + related stuff is extracted and contained
 + in line comment moves to class API  docs